lib/govuk_docker/setup/homebrew_dependencies.rb

```diff
@@ -0,0 +1,8 @@
+require_relative "./base"
+require_relative "../doctor/checkup"
+
+class GovukDocker::Setup::HomebrewDependencies < GovukDocker::Setup::Base
+  def call
+    system("brew bundle --file=Brewfile")
```

Comment:
This doesn't need to specify `--file=Brewfile` as Brewfile is the default file path.

This was leftover from when I tried to be too clever and use `GovukDocker::Paths.govuk_docker_dir` in case people ran `govuk-docker setup` from a location other than the root of the repo. But I think I was over-engineering - the README instructions say to run `govuk-docker setup` from the root - so I backed those changes out.

Should I remove this or is it still useful somehow?

Reply:
This seems like a difficult call, since it the rest of the command does seem to work in other directories. I reckon this level of graceful behaviour isn't necessary, but I don't think we can just lose the functionality in one place and not in others.

Reply:
I don't understand what you want me to do, sorry. 😟Shall we keep it as it is?

Reply:
Sorry for the confusion: I was trying to say that we should still make sure this works when people run it in directories outside of govuk-docker, since that appears to be supported by the other parts of the setup command. This seems unnecessary and we can probably lose it, but it would be confusing to do so in this PR. Does that make sense?

Reply:
Having re-read this three times, yep, this makes sense.

We should make it work from anywhere in this PR. I'll (try to) do that (but may need some help with the tests if my intial experiments were anything to go by). And in a separate PR we can potenetially remove the logic that does it?

Reply:
I pushed a commit to do the path thing, and squashed the other tiny changes: now running `govuk-docker setup` from anywhere works fine. 🎉 I'll make this into a Real PR now.